src/Compiler/Facilities/LanguageFeatures.fsi

```diff
@@ -40,6 +40,8 @@ type LanguageFeature =
     | ReallyLongLists
     | ErrorOnDeprecatedRequireQualifiedAccess
     | LowercaseDUWhenRequireQualifiedAccess
+    | InterfacesWithAbstractStaticMembers
+    | SelfTypeConstraints
```

Comment:
To be honest, I don't know what to expect from this. But since I don't know what this is supposed to mean, I don't have a better name :D

Reply:
Those are language features, the corresponding "user-facing" names are:
```fsharp
featureInterfacesWithAbstractStaticMembers,"static abstract interface members"
featureSelfTypeConstraints,"self type constraints"
```
Those are used like "feature flags", and tied to a specific language version. We use them to guard certain implementations in code.